packages/help-center/src/components/help-center-search.tsx

```diff
@@ -50,11 +56,13 @@ export const HelpCenterSearch = ( { onSearchChange, currentRoute }: HelpCenterSe
 
 	// Search query can be a query param, if the user searches or clears the search field
 	// we need to keep the query param up-to-date with that
+	// Search term is used to set a search term before opening help center
 	useEffect( () => {
-		if ( query ) {
+		const hasSearchQuery = query || searchTerm;
```

Comment:
I don't think we need to change this effect. `searchTerm` shouldn't change the URL, only `searchQuery`.

Reply:
You're right. Thank you!